tests/test_selection/conftest.py

```diff
@@ -55,3 +55,13 @@ def df_test_num_cat():
     X = df[["var_A", "var_B", "var_C", "var_D"]]
     y = df["target"]
     return X, y
+
+
+@pytest.fixture(scope="module", name="dummy_method")
```

Comment:
@TremaMiguel this PR looks good to merge to me. The only request, could we change the name of this function to something more representative of what the function does? 

In case we want to use this function for other methods, we get an intuition through the name of what the function does. :)